README.md

```diff
@@ -1,127 +1,98 @@
+See [docs](https://pingcap.com/docs/) or
+[docs-cn](https://pingcap.com/docs-cn/) for user documents. Also you can get
+help in [AskTUG](https://asktug.com/) if you met any problem.
 
-+ Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
+## TiDB Developer Group
 
-+ Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
+### Communicating
 
+The [communication](./communicating.md) page lists communication channels like
+chat, social medias, etc.
 
-+ Reddit: [https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
+For more specific topics, try to join
+[https://pingcap.com/tidbslack](https://pingcap.com/tidbslack) and discuss with others.
 
-## How Can I Help?
+### Governance
 
-+ Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+TiDB has the following types of groups that are officially supported:
 
-+ To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17/), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a
+  part of the TiDB. SIGs must have open and transparent proceedings. Anyone is
+  welcome to participate and contribute provided they follow the TiDB [Code of
+  Conduct](./CODE_OF_CONDUCT.md). The purpose of a SIG is to own and develop a
+  part of TiDB.
 
-+ Start with improving the documentation. You can search [open issues](https://github.com/pingcap/docs/issues) to find inspiration for what to work on.
+* **Working Groups (WGs)** are temporary groups that are formed to address
+  issues that cross SIG boundaries. Working groups do not own any code or other
+  long term artifacts. Working groups can report back and act through involved
+  SIGs.
 
-+ There's always code that can be clarified and variables or functions that can be renamed or commented.
+### How to contribute
 
-+ There's always a need for more test coverage. You can start with an area that has [low coverage](https://codecov.io/gh/pingcap/tidb).
+Contributions are welcomed and greatly appreciated. See
+[CONTRIBUTING.md](./CONTRIBUTING.md)
+for details on submitting patches and the contribution workflow.
 
-+ Give a talk at a local meetup or conference.
+You can start by finding an existing issue with the
+[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+label in the tidb repo. These issues are well suited for new contributors. BTW,
+there's always a need for more test coverage. You can also start with an area
+that has [low coverage](https://codecov.io/gh/pingcap/tidb).
 
-## Learn to Use TiDB
+You can also join one of the [active working groups](./working-groups) to
+participate in a community activity. Start attending working group meetings and
```

Comment:
```suggestion
participate in a community activity. Start with attending working group meetings or
```

Reply:
> Start attending working group meetings and

This sentence feels disconnected from the context. So I assume it's a suggestion on how to join working groups.